addons/website/static/src/scss/website.scss

```diff
@@ -36,7 +36,16 @@ $-seen-urls: ();
     @include print-variable('has-customized-13-0-color-system', $o-has-customized-13-0-color-system);
 
     // 1) Handle default values
+    @include print-variable('paragraph-margin-bottom', $paragraph-margin-top);
```

Comment:
Defaulting to its own value now, and set default top to 0.

Reply:
Note that it seems that since <https://github.com/odoo/odoo/commit/22e5a56696928273392e8f92cbd081e50a2dee02#diff-ffb61e86e6b8297ef8997f9c605de14de896b54161c032d88a80bbaeac4f89beR757> a value of 0 is applied when emptying the number. Maybe I should add another parameter to choose "reset" instead of "0".